.eslintrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  extends: ["@carforyou/eslint-config"],
```

Comment:
I interpret that this extends all rules we have, but it's probably just the base ones. Not sure.. might be just me stumbling here

Reply:
It's the base javascript/typescript set of rules that is the default. Nothing that the docs can't fix I'd say.